dds_web/__init__.py

```diff
@@ -323,6 +328,7 @@ def load_user(user_id):
         app.cli.add_command(send_usage)
         app.cli.add_command(collect_stats)
         app.cli.add_command(monitor_usage)
+        app.cli.add_command(run_redis_worker)
```

Comment:
this should probably not be under "Add flask commands - cronjobs" but under "Add flask commands - general"